From 0c9fd02cce79ca7408cb4572b9bb5d040f16d649 Mon Sep 17 00:00:00 2001
Message-ID: <0c9fd02cce79ca7408cb4572b9bb5d040f16d649.1707646257.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1707646257.git.sharlatanus@gmail.com>
References: <cover.1707646257.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 10 Feb 2024 16:42:16 +0000
Subject: [PATCH 18/30] gnu: go-golang-org-x-xerrors: Move to (gnu
 packages-build).

* gnu/packages/golang.scm (go-golang-org-x-xerrors): Move from here ...
* gnu/packages/golang-build.scm: ... to here.

Change-Id: I68c9aa1607d71d08eb7ea4e5caf27121cdd0df4c
---
 gnu/packages/golang-build.scm | 25 +++++++++++++++++++++++++
 gnu/packages/golang.scm       | 24 ------------------------
 2 files changed, 25 insertions(+), 24 deletions(-)

diff --git a/gnu/packages/golang-build.scm b/gnu/packages/golang-build.scm
index a8e97e7acb..fed606e8fb 100644
--- a/gnu/packages/golang-build.scm
+++ b/gnu/packages/golang-build.scm
@@ -1,5 +1,6 @@
 ;;; GNU Guix --- Functional package management for GNU
 ;;; Copyright © 2020 Leo Famulari <leo@famulari.name>
+;;; Copyright © 2020 Ryan Prior <rprior@protonmail.com>
 ;;; Copyright © 2021 Ludovic Courtès <ludo@gnu.org>
 ;;; Copyright © 2021 hackeryarn <artemchernyak@gmail.com>
 ;;; Copyright © 2023 Artyom V. Poptsov <poptsov.artyom@gmail.com>
@@ -195,6 +196,30 @@ (define-public go-golang-org-x-sys
 support for low-level interaction with the operating system.")
       (license license:bsd-3))))
 
+(define-public go-golang-org-x-xerrors
+  (let ((commit "5ec99f83aff198f5fbd629d6c8d8eb38a04218ca")
+        (revision "0"))
+    (package
+      (name "go-golang-org-x-xerrors")
+      (version (git-version "0.0.0" revision commit))
+      (source
+       (origin
+         (method git-fetch)
+         (uri (git-reference
+               (url "https://go.googlesource.com/xerrors")
+               (commit commit)))
+         (file-name (git-file-name name version))
+         (sha256
+          (base32 "1dbzc3gmf2haazpv7cgmv97rq40g2xzwbglc17vas8dwhgwgwrzb"))))
+      (build-system go-build-system)
+      (arguments
+       '(#:import-path "golang.org/x/xerrors"))
+      (home-page "https://godoc.org/golang.org/x/xerrors")
+      (synopsis "Go 1.13 error values")
+      (description "This package holds the transition packages for the new Go
+1.13 error values.")
+      (license license:bsd-3))))
+
 ;;;
 ;;; Avoid adding new packages to the end of this file. To reduce the chances
 ;;; of a merge conflict, place them above by existing packages with similar
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index e7b98028a5..68b905ff8b 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -3675,30 +3675,6 @@ (define-public go-golang-org-x-oauth2
  spec in Go.")
       (license license:bsd-3))))
 
-(define-public go-golang-org-x-xerrors
-  (let ((commit "5ec99f83aff198f5fbd629d6c8d8eb38a04218ca")
-        (revision "0"))
-    (package
-      (name "go-golang-org-x-xerrors")
-      (version (git-version "0.0.0" revision commit))
-      (source (origin
-                (method git-fetch)
-                (uri (git-reference
-                      (url "https://go.googlesource.com/xerrors")
-                      (commit commit)))
-                (file-name (git-file-name name version))
-                (sha256
-                 (base32
-                  "1dbzc3gmf2haazpv7cgmv97rq40g2xzwbglc17vas8dwhgwgwrzb"))))
-      (build-system go-build-system)
-      (arguments
-       '(#:import-path "golang.org/x/xerrors"))
-      (synopsis "Go 1.13 error values")
-      (description
-       "This package holds the transition packages for the new Go 1.13 error values.")
-      (home-page "https://godoc.org/golang.org/x/xerrors")
-      (license license:bsd-3))))
-
 (define-public go-github-com-hashicorp-go-uuid
   (package
     (name "go-github-com-hashicorp-go-uuid")
-- 
2.41.0

